nncf/experimental/post_training/graph/model_transformer.py

```diff
@@ -0,0 +1,125 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List, TypeVar
+
+from nncf.common.graph.model_transformer import ModelTransformer
+from nncf.common.graph.transformations.commands import TransformationCommand
+from nncf.common.graph.transformations.layout import TransformationLayout
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPointsContainer
+
+ModelType = TypeVar('ModelType')
+
+
+# pylint: disable=no-member
+class ModelTransformerMeta(ModelTransformer):
```

Comment:
`Meta` in Python should be reserved for metaclasses, which this class is not. Perhaps this should rather be called a `StaticModelTransformerBase` to be more precise on the purpose of this class. Consider making this an ABC since it has unimplemented methods.

Reply:
Renamed, converted into ABC